aea/cli/add_key.py

```diff
@@ -50,22 +50,26 @@
 @click.argument(
     "file", metavar="FILE", type=key_file_argument, required=False,
 )
+@click.argument(
+    "password", metavar="PASSWORD", type=str, default=None, required=False,
```

Comment:
Did I do this right @marcofavorito ?

Reply:
I think that passing the password as an argument in plaintext is a bad practice. As a temporary (but *very* temporary) solution, it is OK. 

Apart from that. looks good. You may want to have a `click.option` instead of an argument: `--password/-p`.

Reply:
I would have used `getpass` instead:
https://docs.python.org/3/library/getpass.html#getpass.getpass

And an optional flag (default to false) to trigger the prompt (e.g. `--use-password`).

Reply:
https://click.palletsprojects.com/en/7.x/options/